CMakeLists.txt

```diff
@@ -145,7 +145,7 @@ if(BUILD_LIBOPAE_CXX)
   add_subdirectory(libopaecxx)
 endif()
 
-option(BUILD_LIBOPAE_PY "Enable building of Python bindings around libopae-cxx-core." ON)
+option(BUILD_LIBOPAE_PY "Enable building of Python bindings around libopae-cxx-core." OFF)
 mark_as_advanced(BUILD_LIBOPAE_PY)
```

Comment:
why is this turned off with your change?

Reply:
in order to run DCP regression. Will change it back.